stable/external-dns/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Deploys external-dns into a kubernetes cluster
```

Comment:
Let's use a better description here. Maybe something like the one on the repo itself:

> Configure external DNS servers (AWS Route53, Google CloudDNS and others) for Kubernetes Ingresses and Services